shell/browser/api/electron_api_app.cc

```diff
@@ -595,11 +595,10 @@ App::App() {
       ->set_delegate(this);
   Browser::Get()->AddObserver(this);
 
-  base::ProcessId pid = base::GetCurrentProcId();
   auto process_metric = std::make_unique<electron::ProcessMetric>(
       content::PROCESS_TYPE_BROWSER, base::GetCurrentProcessHandle(),
       base::ProcessMetrics::CreateCurrentProcessMetrics());
-  app_metrics_[pid] = std::move(process_metric);
+  app_metrics_[0] = std::move(process_metric);
```

Comment:
Though `0` is guaranteed not to be returned in `GetID`, using `content::ChildProcessHost::kInvalidUniqueID` would make the code more future proof.